README.md

```diff
@@ -67,9 +67,21 @@ to revise it in response.
 * Build consensus and integrate feedback. RFCs that have broad support
 are much more likely to make progress than those that don't receive any
 comments.
-* Eventually, somebody on the [core team] will either accept the RFC by
-merging the pull request and assigning the RFC a number, at which point
-the RFC is 'active', or reject it by closing the pull request.
+* Eventually, the [core team] will decide whether the RFC is a candidate
```

Comment:
@dgeb I thought this was supposed to be the subteam that puts an RFC into FCP, then the core team have seven days to respond. The intent is that the subteams can iterate on RFCs and FCP is when core should look up and be aware of what is happening.